postgresql-ballerina/tests/call-procedures-test.bal

```diff
@@ -0,0 +1,1755 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/sql;
+import ballerina/test;
+import ballerina/time;
+
+public type StringDataForCall record {
+    string char_type;
+    string varchar_type;
+    string text_type;
+    string name_type;
+};
+
+@test:Config {
+    groups: ["procedures"]
+}
+function testProcedureQueryWithSingleData() {
+    int row_id = 1;
+    sql:ParameterizedCallQuery callQuery = `
+        select * from singleSelectProcedure(${row_id});
+    `;
+
+    sql:ProcedureCallResult ret = callProcedure(callQuery, proceduresDatabase, [StringDataForCall]);
+    stream<record{}, sql:Error>? qResult = ret.queryResult;
+    if (qResult is ()) {
+        test:assertFail("Empty result set returned.");
+    } else {
+        record {|record {} value;|}? data = checkpanic qResult.next();
```

Comment:
shall we use `check` instead of `checkpanic` and returns error

Reply:
I will change

Reply:
I changed. Commit Id is 2cbf8db5f212c76d3f97ac3049e5ab0631ecd60c